src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -1202,9 +1258,9 @@ public void testPutOrdersByIdToChangeStatusToOpenWithEmptyPoLines() throws Excep
   private void verifyReceiptStatusChangedTo(String expectedStatus) {
     List<JsonObject> polUpdates = MockServer.serverRqRs.get(PO_LINES, HttpMethod.PUT);
     assertNotNull(polUpdates);
-    // check with filter last 2 putted polines
+    // check receipt status for the last 2 putted polines
     for (JsonObject jsonObj : polUpdates.subList(polUpdates.size() - 2, polUpdates.size())) {
-      assertTrue(jsonObj.getString(RECEIPT_STATUS).equals(expectedStatus));
+      assertEquals(jsonObj.getString(RECEIPT_STATUS), expectedStatus);
```

Comment:
Please change order of the arguments according to method definition:
`static public void assertEquals(Object expected, Object actual)`

Reply:
fixed arguments order